tutor/src/models/user/terms.js

```diff
@@ -25,7 +24,7 @@ class UserTerms extends BaseModel {
 
   constructor(attrs) {
     super(attrs);
-    if (this.user.terms_signatures_needed) { this.fetch(); }
+    observe(this.user, 'terms_signatures_needed', this.fetchIfNeeded);
   }
```

Comment:
this is the bug.  the user model is created before it's bootstrapped, which causes terms_signatures_needed to be false initially, then later it'll be set to true when the app is bootstrapped